### PR TITLE
Make --optimize the default #439

### PR DESCRIPTION
**Pull request by mjobuda**
--optimize is now an option,--optimize=true is default

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -152,14 +152,18 @@ pub fn main() {
...
+                .use_delimiter(false)
+                .takes_value(true),
         )
         .get_matches();
 
     let input_file = matches.value_of("input").unwrap();
     let output_dir = matches.value_of("output-dir").unwrap();
     let overwrite = matches.is_present("overwrite");
-    let optimize = matches.is_present("optimize");
+    let optimize = matches.value_of("optimize")==Some("true");
```
Comment (g-r-a-n-t) [inline src/main.rs:166]: The formatter would like spaces around `==`.

**Review by g-r-a-n-t — APPROVED** [review]
Looks great. Thanks for sending this. You can run `cargo fmt` to fix the formatting or do it manually.

File: src/main.rs (hunk 1 above)
Comment (mjobuda) [reply src/main.rs]: Thanks! The formatter is right. It should be with spaces. I corrected it. And updated the docs at https://github.com/mjobuda/fe/blob/master/docs/src/quickstart/installation.md
Should I do another pull request for the updated doc or how is the procedure?

File: newsfragments/439.feature.rst [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Enable the optimizer by default. The optimizer can still be disabled
+by supplying `--optimize=false` as an argument.
```
Comment (cburgdorf) [inline newsfragments/439.feature.rst:2]: @mjobuda I added this file to your PR. It ensures your change will be mentioned in the upcoming release notes. I will wait for CI to finish and then merge.

**Pull request merged**